modules/sdk-coin-bera/test/resources.ts

```diff
@@ -0,0 +1,85 @@
+
+export const SEND_FUNDS_AMOUNT_ZERO_DATA =
+  '0x391252150000000000000000000000007325a3f7d4f9e86ae62cf742426078c3755730d5000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c0000000000000000000000000000000000000000000000000000000005ec6ab34000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000e000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000041aac0763277264c9be1e79a7b4bf056f3011a545ca2ead9fe76aa6efaa80b35db1b9769bf44693896ca655eb845b8e796f71073096e00f4ab8eddf1c46b8d7e401c00000000000000000000000000000000000000000000000000000000000000';
+
+export const SEND_BERA_LINK_DATA =
+  '0x0dcd7a6c0000000000000000000000007325a3f7d4f9e86ae62cf742426078c3755730d5000000000000000000000000000000000000000000000000016345785d8a0000000000000000000000000000143e1dae4f018ff86051a01d44a1b49b13704056000000000000000000000000000000000000000000000000000000005ec6ab34000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000c00000000000000000000000000000000000000000000000000000000000000041c6f5ab6c29ea92fec02229e8f87981fdb0863bf465f3d4f08df81d66d12d771a588acaf9d2313704dfb120da5e50a3a6089abbe610fca825d1a1fc9f3287ab8a1c00000000000000000000000000000000000000000000000000000000000000';
+
+export const SEND_TOKEN_SEQUENCE_ZERO_DATA =
+  '0x0dcd7a6c0000000000000000000000007325a3f7d4f9e86ae62cf742426078c3755730d5000000000000000000000000000000000000000000000000016345785d8a0000000000000000000000000000143e1dae4f018ff86051a01d44a1b49b13704056000000000000000000000000000000000000000000000000000000005ec6ab34000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c00000000000000000000000000000000000000000000000000000000000000041c92569648e4bb36f8890254998b71359992f6afa97bba96975297cff3272dc402bfae501804561ca3bed0f38057ca6cc896950c50a35322b6062a5f132c653711b00000000000000000000000000000000000000000000000000000000000000';
+
+export const SEND_TOKEN_AMOUNT_ZERO_DATA =
+  '0x0dcd7a6c0000000000000000000000007325a3f7d4f9e86ae62cf742426078c3755730d50000000000000000000000000000000000000000000000000000000000000000000000000000000000000000143e1dae4f018ff86051a01d44a1b49b13704056000000000000000000000000000000000000000000000000000000005ec6ab34000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000c00000000000000000000000000000000000000000000000000000000000000041f159de6649d213fe6b882444d525ac3f9a7393618d510506f250931a28f6c34b50bab2af7f77e893e281ab3d0f629190d70ab271e491b4b41fa75b4dbf97cd191c00000000000000000000000000000000000000000000000000000000000000';
```

Comment:
I think we can remove token related data as that is not required right now, are we going to onboard berachain tokens?

Reply:
done